packages/libraries/main/src/matcher/dictionary/feedback.ts

```diff
@@ -53,7 +53,8 @@ const getDictionaryWarning = (
   const dictName = match.dictionaryName
   const isAName =
     dictName.toLowerCase().includes('lastnames') ||
-    dictName.toLowerCase().includes('firstnames')
+    dictName.toLowerCase().includes('firstnames') ||
```

Comment:
Checking for "lastnames" and "firstnames" looks unnecessary because they include "names".